Project.toml

```diff
@@ -1,29 +1,39 @@
 name = "AstroImages"
 uuid = "fe3fc30c-9b16-11e9-1c73-17dabf39f4ad"
-authors = ["Mosè Giordano", "Rohit Kumar"]
-version = "0.2.0"
+authors = ["Mosè Giordano", "Rohit Kumar", "William Thompson"]
+version = "0.3.0"
 
 [deps]
+AbstractFFTs = "621f4979-c628-5d54-868e-fcf4e3e8185c"
+AstroAngles = "5c4adb95-c1fc-4c53-b4ea-2a94080c53d2"
+ColorSchemes = "35d6a980-a343-548e-a6ea-1d62b119f2f4"
+DimensionalData = "0703355e-b756-11e9-17c0-8b28908087d0"
 FITSIO = "525bcba6-941b-5504-bd06-fd0dc1a4d2eb"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
-ImageMagick = "6218d12a-5da1-5696-b52f-db25d2ecc6d1"
-Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
-Interact = "c601a237-2ae4-5e1e-952c-7a85b0c7eef1"
+ImageAxes = "2803e5a7-5153-5ecf-9a86-9b4c37f5f5ac"
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+ImageMetadata = "bc367c6b-8a6b-528e-b4bd-a4b897500b49"
+ImageShow = "4e3cecfd-b093-5904-9786-8bbb286a6a31"
+ImageTransformations = "02fcd773-0e25-5acc-982a-7f6622650795"
+MappedArrays = "dbb5928d-eab1-5f90-85c2-b9b0edb7c900"
+PlotUtils = "995b91a9-d308-5afd-9ec6-746e21dbc043"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
-Reproject = "d1dcc2e6-806e-11e9-2897-3f99785db2ae"
```

Comment:
If you need it, `Reproject.jl` [v0.3.1](https://github.com/JuliaAstro/Reproject.jl/releases/tag/v0.3.1) allows the latest version of `Interpolations.jl`

Reply:
Thanks @giordano! I had just run into that issue.

One thing I am wondering is whether it might be better to have the opposite relationship, with Reproject.jl depending on AstroImages.
What are your thoughts?

Reply:
`Reproject.jl` was meant to be a small self-contained low-level package for rotating images (much like its [Python counterpart](https://github.com/astropy/reproject)).  How are you dealing with the rotations?

Reply:
Right now there is nothing that rotates or otherwise transforms images, except for the RGB composite functions that first call reproject.
I was considering separating this out to look something like:
```julia
composechannels([
    astroimg1,
    reproject(astroimg2, wcs(astroimg1,1)),
    reproject(astroimg3, wcs(astroimg1,1)),
],   ...)
```

where `composechannels` is a shortcut for `0.3 imview(img1, cmap=:reds) .+ 0.3 imview(img1, cmap=:greens) .+ ...`

One thing that doesn't yet work is reprojecting one slice of a cube, but it should be possible with all the information now tracked by the `AstroImage` struct.